lib/widgets/modular_widgets/transfer_widgets/send/send_medium.dart

```diff
@@ -321,12 +324,12 @@ class _SendMediumCardState extends State<SendMediumCard> {
   bool _isInputValid(AccountInfo accountInfo) =>
       InputValidators.checkAddress(_recipientController.text) == null &&
       InputValidators.correctValue(
-            _amountController.text,
-            accountInfo.getBalanceWithDecimals(
-              _selectedToken.tokenStandard,
-            ),
-            _selectedToken.decimals,
-          ) ==
+              _amountController.text,
+              accountInfo.getBalance(
+                _selectedToken.tokenStandard,
+              ),
+              _selectedToken.decimals,
+              BigInt.zero) ==
```

Comment:
The UI indicates that 0 is valid but the `Send payment` button can't be pressed.
Also applies to the send_large.dart widget.
<img src="https://github.com/zenon-network/syrius/assets/104875699/ceb8ee9f-7eed-4d5c-ab65-09d3e67d0ab5" width="50%" height="50%">